src/components/shared/InformationIcon.tsx

```diff
@@ -23,6 +23,7 @@ export default function InformationIcon(props: InformationIconProps) {
         offset={offset}
         position={position}
         size={popupSize}
+        hoverable={true}

```

Comment:
nit: this should work too right?
```suggestion
        hoverable

```